### PR TITLE
Fix CLI help bug

### LINKED ISSUE #58: help function from the CLI is not working in the main branch
Trying to execute the command `oqt create-indicator --help` the error 
"AttributeError: 'NoneType' object has no attribute 'upper'" 
occurs instead of printing the help. Same applies for `oqt create-report --help`, but not for all other commands including `oqt create-all-indicators --help`

This bug appears in the main branch

### PR DESCRIPTION
**Pull request by matthiasschaub**
### Description

This bug has been caused due to missing name attribute of custom click
type class (feature_id_field)

### Corresponding issue
Closes #58

### Checklist
- [x] I have updated my branch to `main` (e.g. through `git rebase main`)
- [x] My code follows the [style guide](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#style-guide) and was checked with [pre-commit](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#pre-commit) before committing
- [x] I have added sufficient unit and integration [tests](https://github.com/GIScience/ohsome-quality-analyst/blob/main/docs/development_setup.md#tests)

### CODE REVIEW

File: workers/ohsome_quality_analyst/cli_opts.py
Comment (Hagellach37) [inline workers/ohsome_quality_analyst/cli_opts.py:21]: I think it should be `integer|string`  not inte**R**ger

**Comment by Hagellach37** [comment]
Hey @matthiasschaub ,
approved but please fix the spelling before merging. :)

**Review by Hagellach37 — APPROVED** [review]

File: workers/ohsome_quality_analyst/cli_opts.py
Comment (matthiasschaub) [reply workers/ohsome_quality_analyst/cli_opts.py]: Is fixed. I need your approval again to be able to merge this PR.

**Review by Hagellach37 — APPROVED** [review]

**Pull request merged**
